src/ray/gcs/client_test.cc

```diff
@@ -87,14 +92,16 @@ class TestGcsWithAsio : public TestGcs {
 };
 
 void ObjectAdded(gcs::AsyncGcsClient *client, const UniqueID &id,
-                 std::shared_ptr<ObjectTableDataT> data) {
-  ASSERT_EQ(data->managers, std::vector<std::string>({"A", "B"}));
+                 const std::vector<ObjectTableDataT> &data) {
+  ASSERT_EQ(data.size(), 1);
+  ASSERT_EQ(data[0].managers, std::vector<std::string>({"A", "B"}));
```

Comment:
I imagine (and hope) `ASSERT_EQ(data[0].managers, {"A", "B"});` just works.

Reply:
It does not, I think because of the `ASSERT_EQ` macro:
```
error: macro "ASSERT_EQ" passed 3 arguments, but takes just 2
   ASSERT_EQ(data[0].managers, {"A", "B"});
```